File: SWEA/D2/1945. 간단한 소인수분해/간단한 소인수분해.py
T = int(input())
for test_case in range(1, T + 1):
    answer_list = [0, 0, 0, 0, 0]
    num = int(input())
    while 1:
        if num % 11 == 0:
            answer_list[4] += 1
            num = num // 11
        elif num % 7 == 0:
            answer_list[3] += 1
            num = num // 7
        elif num % 5 == 0:
            answer_list[2] += 1
            num = num // 5
        elif num % 3 == 0:
            answer_list[1] += 1
            num = num // 3
        elif num % 2 == 0:
            answer_list[0] += 1
            num = num // 2
        else:
            print(f'#{test_case} {answer_list[0]} {answer_list[1]} {answer_list[2]} {answer_list[3]} {answer_list[4]}')
            break

File: SWEA/D2/1945. 간단한 소인수분해/간단한 소인수분해.java
import java.util.Scanner;
import java.io.FileInputStream;

class Solution
{
	public static void main(String args[]) throws Exception
	{
		Scanner scanner = new Scanner(System.in);
        int T = scanner.nextInt();
        for (int test_case = 1; test_case <= T; test_case++) {
            int[] answerList = {0, 0, 0, 0, 0};
            int num = scanner.nextInt();
            while (true) {
                if (num % 11 == 0) {
                    answerList[4]++;
                    num /= 11;
                } else if (num % 7 == 0) {
                    answerList[3]++;
                    num /= 7;
                } else if (num % 5 == 0) {
                    answerList[2]++;
                    num /= 5;
                } else if (num % 3 == 0) {
                    answerList[1]++;
                    num /= 3;
                } else if (num % 2 == 0) {
                    answerList[0]++;
                    num /= 2;
                } else {
                    System.out.printf("#%d %d %d %d %d %d\n", test_case, answerList[0], answerList[1], answerList[2], answerList[3], answerList[4]);
                    break;
                }
            }
        }
	}
}